packages/components/src/settings/feature-options-defaults.json

```diff
@@ -197,6 +197,26 @@
       "minValue": 1,
       "maxValue": 5000,
       "values": []
+    },
+    "enableMouseMoveLimitedByIdCheck": {
+      "defaultValue": false,
+      "labelText": "Limit by Id (recommended for Google Docs)",
```

Comment:
can we make this simpler given that it should be only applied to google docs for now?

E.g. maybe a single setting: 

enableMouseMoveOnGoogleDocsId: docs-editor-container

And then we can use this setting both to determine whether or not to auto-enable mouse-move on google docs, and if so, on which element?

Reply:
And then the text displayed to the user for the setting could be:

Enable Mouse Move on Google Docs Element Id:

Reply:
ok, I would change it - but this way id name won't be confugarable (per requirements)

Reply:
@balmas, I believe that our requirements became circual :)
Let's check final requirements

1. We should enable mouseMove on google docs by default
2. We should have ability to limit by id (for now it is fixied to `docs-editor-container`) - via checkbox

Reply:
![image](https://user-images.githubusercontent.com/12377640/78647440-f45cc180-78fd-11ea-9eed-f3dd9e4e8988.png)

Reply:
> @balmas, I believe that our requirements became circual :)
> Let's check final requirements
> 
> 1. We should enable mouseMove on google docs by default
> 2. We should have ability to limit by id (for now it is fixied to `docs-editor-container`) - via checkbox
> 3. We could enable mouseMove on any site (it was done before)

oh dear, I think my language was confusing here.

I meant for the setting value to be the id of the element it should be limited to

i.e. default setting value would be "docs-editor-container"

And it would be displayed as such, rather than as a checkbox toggle.